packages/App/src/balance/wallet/Wallet.tsx

```diff
@@ -16,12 +16,26 @@ import iconShoppingSource from '../../../Pics/categories/shop-bag.png';
 import iconRestaurantSource from '../../../Pics/categories/restaurant.png';
 import iconSalarySource from '../../../Pics/categories/money.png';
 
-const wallets = [
+interface walletInfo {
+  key: string;
+  color: string;
+  walletTitle: string;
+  walletAmount: number;
+  transactions: Array<{
+    type: string;
```

Comment:
Looks like it can be switched to a string literal union (like `'expenses' | 'incomes'`) or an enum